HaxeManual/02-types.tex

```diff
@@ -940,6 +940,12 @@ \subsection{Enum abstracts}
 	\item Defining fields requires less syntax.
 \end{itemize}
 
+\since{4.0.0}
+
+Since Haxe 4, it is possible to declare enum abstracts without using the \expr{:enum} metadata, instead using the more natural syntax \expr{enum abstract}. Additionally, if the underlying type is \expr{String} or \expr{Int}, the values for the enum cases can be omitted and are deduced by the compiler. For \expr{Int} abstracts, the deduced values increment the last user-defined value, or start at zero if no value was declared yet:
```

Comment:
The example shows both Int and String abstracts, but this paragraph only mentions what the compiler does with Int abstracts.

Reply:
The code example shows the String values in comments next to the cases. Should I add it in the text as well?

Reply:
Well that's my point. The intro text with the colon makes it sound like the example is specific to Int, but then it also shows String. Would be better to quickly mention the latter too.